app/logic/getSLInstructorTableData.py

```diff
@@ -0,0 +1,20 @@
+from app.models.course import Course
+from app.models.user import User
+from app.models.courseInstructor import CourseInstructor
+
+def getProposalData(user):
+    """Returns dictionary with data used to populate SL proposal table"""
+    courses = (Course.select()
+                     .where(CourseInstructor.user==user)
+                     .join(CourseInstructor))

```

Comment:
Add an explicit order_by

Reply:
I added a .order_by(Course.id) is that what you meant? or should I be ordering by something else?

Reply:
Depends on what you want to order by. course ID is fine, if you don't have another preference, but think about what it will look like a few years down the road. Which courses do you want showing first?